src/styles/blocks/main/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+  margin-left: 48px;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    max-width: 166px;
+    margin-bottom: 4px;
+
+    color: var(--blue-color);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+
+    min-height: 134px;
+    max-width: 160px;
+  }
+
+  &__subtitle {
+    color: var(--grey-color);
+    font-size: 10px;
+    margin-bottom: 17px;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--grey-color);
+
+    &-number {
+      font-weight: 700;
+      font-size: 16px;
+      color: var(--blue-color);
+    }
+
+    &-container {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+
+      margin-bottom: 16px;
+    }
+  }
+
+  &__link {
+    box-sizing: border-box;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    padding: 10px 57px 11px 55px;
+    background-color: #00acdc;
+    border: 1px solid rgba(0, 0, 0, 0);
+    border-radius: 5px;
+
+    &:hover {
+      background-color: #fff;
+      color: #00acdc;
+      border-color: rgba(0,172,220,1);
+    }
+  }
+
+  &__stars {
+    display: flex;
+    flex-direction: row;
+  }
```

Comment:
that should be style for stars block, not element, and they should be in stars.scss